make_html.py

```diff
@@ -163,9 +138,8 @@ class Mathlib3FileData:
     mathlib3_import: List[str]
     status: port_status_yaml.PortStatusEntry
     lines: Optional[int]
-    labels: Optional[List[dict[str, str]]]
-    dependents: Optional[List['Mathlib3FileData']] = None
-    dependencies: Optional[List['Mathlib3FileData']] = None
+    dependents: List['Mathlib3FileData'] = field(default_factory=list)
+    dependencies: List['Mathlib3FileData'] = field(default_factory=list)
```

Comment:
This is inaccurate; these can be `None` if the file doesn't have any data at all.

Reply:
Hmm, I got an error in `file.j2` at l.259 (`{{ progress.bars([data] + data.dependencies) }}`) that `data.dependencies` is `None` but expected to be `list`. I'll fix that another way then.

Reply:
This usually means that you need to `git pull` in your `mathlib` directory.

But ideally we would add a `{% if data.dependencies is none %}some warning message{%else%} the progress bar {% endif %}`

Reply:
> This usually means that you need to `git pull` in your `mathlib` directory.
Ah